### PR TITLE
Added java_vendor support. Added new screenshot and updated documentation

### PR DESCRIPTION
**Pull request by justnoxx**
Should be merged only if we're decided to have configurable JAVA_VENDOR environment variable support.

### CODE REVIEW

File: pages/EC-WebLogic_help.xml [hunk 1]
```diff
@@ -200,6 +200,10 @@
                         <td>JAVA_HOME environment variable. It should be used when java is not in the path, or different java need to be used.</td>
```
Comment (urvashisingh) [inline pages/EC-WebLogic_help.xml]: AVA_HOME environment variable. It should be used when java is not in the path, or a different JDK needs to be used.

File: pages/EC-WebLogic_help.xml [hunk 2]
```diff
@@ -200,6 +200,10 @@
...
                     </tr>
                     <tr>
+                      <td>JAVA_VENDOR</td>
+                      <td>JAVA_VENDOR environment variable. It should be used when java vendor need to be specified.</td>
```
Comment (urvashisingh) [inline pages/EC-WebLogic_help.xml]: JAVA_VENDOR environment variable. It should be used to specify the Java vendor value when setting the JAVA_HOME parameter to point to a specific JDK.

Comment (justnoxx) [inline pages/EC-WebLogic_help.xml]: Done.

File: pages/EC-WebLogic_help.xml (hunk 1 above)
Comment (justnoxx) [inline pages/EC-WebLogic_help.xml]: Done.

**Pull request merged**